.github/ISSUE_TEMPLATE/rdsm-ruby-client-issue-template.md

```diff
@@ -0,0 +1,49 @@
+---
+name: RDSM Ruby Client Issue Template
+about: Template for new Issues for the RD Station Ruby Client Project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# Prerequisites
+
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
```

Comment:
We can remove this part: `(for multi-repository projects)`. What do you think?

Reply:
Nice suggestion ;)